### DISCUSSION TITLE
Cesium-terrain and Geoserver

### DISCUSSION THREAD

**Discussion by gscaduto** [TerriaJS/terriajs#6631] [Q&A]
Hi,

how can I use "type": "cesium-terrain" with geoserver layers?

Thanks
Gabriele

**Comment by hardreddata** [comment]
You will need to publish at least two layers for this. 

One will be the cesium terrain layer. The other will be a WMS or WFS or similar from GeoServer. See `WebMapServiceCatalogItem` or `WebMapServiceCatalogGroup`.

Cheers.

**Reply by gscaduto** [reply]
i want use my custom cesium-terrain, work with terrain in cesium ION but not like geoserver DSM layer .
